enterprise/lms_api.py

```diff
@@ -24,24 +26,36 @@
     CourseEnrollment = None
 
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
@e-kolpakov [Other files in the repo also use the `LOGGER` name](https://github.com/edx/edx-enterprise/search?utf8=%E2%9C%93&q=logger), so I believe this is up for debate.

@brittneyexline @haikuginger Would you like to weigh in?  I'm not fussed either way, but would like to be consistent.